### ISSUE TITLE
better escaping

### ISSUE THREAD

**Issue by norcross** [reaktivstudios/gp-pro-freeform-style#2]
I think following these guidelines would potentially be a good approach: https://make.wordpress.org/themes/2015/02/10/custom-css-boxes-in-themes/

**Comment by jjeaton** [comment]
Yeah, I think we can go without CSS Tidy and just sanitize on save like they mention.

**Comment by norcross** [comment]
👍
